packages/upset/src/utils/exports.ts

```diff
@@ -0,0 +1,66 @@
+import { AccessibleData, Row, Rows, isRowAggregate } from "@visdesignlab/upset2-core";
+
+export const exportState = (provenance: any, data?: any, rows?: Rows) => {
+  let filename = `upset_state_${new Date().toJSON().slice(0,10)}`;
+  let dataObj = provenance.getState();
+
+  if (data && rows) {
+    dataObj = {...dataObj, 'rawData': data, 'processedData': rows, 'accessibleProcessedData': getAccessibleData(rows)};
+    filename = `upset_state_data_${new Date().toJSON().slice(0,10)}`;
+  } else if (data) {
+    dataObj = {...dataObj, 'rawData': data};
+    filename = `upset_state_raw_data_${new Date().toJSON().slice(0,10)}`;
+  }
+
+  const json = JSON.stringify(dataObj, null, 2);
+
+  downloadJSON(filename, json);
+}
+
+export const exportRawData = (data: any) => {
+    const filename = `upset_data_raw_${new Date().toJSON().slice(0,10)}`;
+    const json = JSON.stringify(data, null, 2);
+    downloadJSON(filename, json);
+}
+
+export const exportProcessedData = (rows: Rows, accessible?: boolean) => {
+    const filename = `upset_data_${new Date().toJSON().slice(0,10)}`;
+    let data: Rows | AccessibleData = (accessible) ? getAccessibleData(rows) : rows;
+
+    const json = JSON.stringify(data, null, 2);
+
+    downloadJSON(filename, json);
+}
```

Comment:
Are these used in your new 2 option menu?

Reply:
They aren't. I left them as an additional util function if we do decide to add just the data export.